graphql/schema.graphql

```diff
@@ -1184,17 +1181,6 @@ type ProjectEventLogEntry {
   after: ProjectEventSettings
 }
 
-type RepoEvents {
```

Comment:
the repo schemas don't seem to be typed more strongly than the project ones, so why does this switch help?

Reply:
They actually _are_ more strongly typed. The [RepoRef](https://github.com/evergreen-ci/evergreen/blob/9aff211951ec3109842c8abc67d889bd8538070b/graphql/schema.graphql#L1329) struct has a lot of non-nullable fields that are nullable in [Project](https://github.com/evergreen-ci/evergreen/blob/9aff211951ec3109842c8abc67d889bd8538070b/graphql/schema.graphql#L1276). This could cause errors if, for example, a new field is added; previous events would have this field as `null`, but the schema would indicate that the field is non-nullable since this is the case for current projects (as opposed to historic project states).

Reply:
thank you for explaining it, I understand better now. 